pkgs/development/python-modules/pre-commit/default.nix

```diff
@@ -0,0 +1,47 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, recursivePthLoader
+, pyaml
+, identify
+, nodeenv
+, virtualenv
+, aspy-yaml
+, six
+, cached-property
+, importlib-resources
+, importlib-metadata
+, toml
+, cfgv
+, python36
+}:
+
+buildPythonPackage rec {
```

Comment:
Is this a library or an application? Please read https://nixos.org/nixpkgs/manual/#building-packages-and-applications.